ttnn/cpp/ttnn/operations/normalization/layernorm/device/multi_core/layernorm_op_multi_core.cpp

```diff
@@ -425,6 +425,9 @@ operation::ProgramWithCallbacks layernorm_multi_core_sharded(
     if (fp32_dest_acc_en) {
         TT_ASSERT(subblock_wt <= 4, "subblock width must less than 4 in fp32 mode");
     }
+    else {
+        TT_ASSERT(subblock_wt <= 8, "subblock width must less than 8");
```

Comment:
Lets use TT_FATAL instead. TT_ASSERT is still silent in release

Reply:
Done, also added conditions for dst_full_sync_en == false and improved error messages.